apps/web/pages/team/[slug].tsx

```diff
@@ -161,14 +159,8 @@ function TeamPage({ team, isUnpublished, markdownStrippedBio, isValidOrgDomain }
       <div className="space-y-6" data-testid="event-types">
         <div className="overflow-hidden rounded-sm border dark:border-gray-900">
           <div className="text-muted p-8 text-center">
-            <h2 className="font-cal text-emphasis mb-2 text-3xl">
-              {" " + t("org_no_teams_yet", { defaultValue: "This organization has no teams yet" })}
-            </h2>
-            <p className="text-emphasis mx-auto max-w-md">
-              {t("org_no_teams_yet_description", {
-                defaultValue: "If you are an administrator, be sure to create teams to be shown here.",
-              })}
-            </p>
+            <h2 className="font-cal text-emphasis mb-2 text-3xl">{" " + t("org_no_teams_yet")}</h2>
+            <p className="text-emphasis mx-auto max-w-md">{t("org_no_teams_yet_description")}</p>
```

Comment:
Removed default values introduced recently as 99% of the extra page size came from logo not i18n